data/components/application-ui-toggles.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Toggles Group
+emoji: ⇌
```

Comment:
Should be using an emoji here, could be something like:

- 🎛
- 🎚
- 🕹

It doesn't have to be super accurate to what the collection is.